examples/gateway/openai/config.yaml

```diff
@@ -1,4 +1,4 @@
-routes:
```

Comment:
Can we show a deprecation warning if we find the following fields?

- `routes`
- `route_type`

Reply:
I think we can insert a warning in `_load_route_config`.

Reply:
Added the warning and test :)